### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by raghav6202**
This is a P.R for the landing page of my component library

### CODE REVIEW

File: landingPage/index.html [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+<body>
+    
+    <nav class="navigation-menu">
+
+<div class="left-side-nav"> 
+
+<img class="logo-component" src="images\logo.PNG" alt="logo image">
+<a class="heading" href="http://">JatayU.I</a>
+
+</div>
```
Comment (nyyshaaa) [inline landingPage/index.html:20]: You can do proper code formatting by using ctrl shift p ,if you have downloaded the extension for that in vs code,to show the indentation and nesting.

File: landingPage/landingPage.css [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+    align-items: center;
+}
+
+.right-side-nav
+{
+
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
```
Comment (nightlight-wander) [inline landingPage/landingPage.css:54]: in the nav container use have used space-evenly,also in the child elements also justify-content has been set to space between.so you can just give margin right auto to left part of nav and margin left auto to right part of nav and keep the parent nav container as flex without giving it any justify content.Or I guess even if you have given space between in parent container or any other similar kind of space property ,you don't need to mention in child again.

**Review by JowelTisso — APPROVED** [review]
If you are using rem, then you can set the html font-size to 62.5% so that your rems can be in a multiple of 10px for your better understanding and calculation. Rest is good.

**Review by nyyshaaa — COMMENTED** [review]
Only these two suggestions from my side.

**Pull request merged**